deps/wazuh_testing/wazuh_testing/modules/aws/constants.py

```diff
@@ -11,11 +11,13 @@
 VPC_FLOW_LOGS = "vpcflowlogs"
 FLOW_LOG_ID = "fl-0754d951c16f517fa"
 CONFIG = 'Config'
+ELASTICLOADBALANCING = 'elasticloadbalancing'
```

Comment:
Shouldn't it be ELASTIC_LOAD_BALANCING to follow the standard with the other constants and pep8?